qa/config/platforms.json

```diff
@@ -6,12 +6,8 @@
     "centos-7": { "box": "elastic/centos-7-x86_64", "type": "redhat" },
     "oel-6": { "box": "elastic/oraclelinux-6-x86_64", "type": "redhat" },
     "oel-7": { "box": "elastic/oraclelinux-7-x86_64", "type": "redhat" },
-    "fedora-28": { "box": "elastic/fedora-28-x86_64", "type": "redhat", "experimental": true },
```

Comment:
are these changes to drop old platforms  in scope? If so, I'd like to know why. I would assume that we would merely include later versions, instead of dropping these entirely.

Reply:
They're not. I'll reinstate them, and bring the removal up in a separate PR, or at least spark a discussion as to whether we want to keep them - they're not run in CI right now, and I was trying to figure it if it was the cause behind test failures. (Actually https://github.com/hashicorp/vagrant/issues/12265 is the cause)